lib/reporters.js

```diff
@@ -26,22 +28,32 @@ class DataDogReporter {
      * @param {string} [apiKey]
      * @param {string} [appKey]
      * @param {string} [apiHost]
+     * @param {string} [customServerURL]
      */
-    constructor(apiKey, appKey, apiHost) {
+    constructor(apiKey, appKey, apiHost, customServerURL) {
         apiKey = apiKey || process.env.DATADOG_API_KEY;
         appKey = appKey || process.env.DATADOG_APP_KEY;
         this.apiHost = apiHost || process.env.DATADOG_API_HOST;
+        this.customServerURL = customServerURL || process.env.CUSTOM_SERVER_URL;
 
         if (!apiKey) {
             throw new Error('DATADOG_API_KEY environment variable not set');
         }
 
-        const configuration = datadogApiClient.client.createConfiguration({
+        // If baseServer is undefined, the Datadog SDK will use the hardcoded Datadog servers
+        // eslint-disable-next-line max-len
+        // See: https://github.com/DataDog/datadog-api-client-typescript/blob/v1.6.0/packages/datadog-api-client-common/servers.ts#L58
```

Comment:
Please configure the `ignoreUrls` option in ESLint instead of adding a single-line exception here.

i.e. this line in `.eslintrc.json`: https://github.com/dbader/node-datadog-metrics/blob/2c4222c49218904190a87a570447e7aa81e725f9/.eslintrc.json#L27

…could be something like:

```js
"max-len": ["error", { "code": 110, "ignoreUrls": true }],
```

(Docs here: https://eslint.org/docs/latest/rules/max-len#options)